store/sqlstore/store.go

```diff
@@ -179,24 +181,57 @@ func New(settings model.SqlSettings, metrics einterfaces.MetricsInterface) *SqlS
 
 	store.initConnection()
 
-	if *settings.DriverName == model.DatabaseDriverPostgres {
-		ver, err := store.GetDbVersion(true)
-		if err != nil {
-			mlog.Critical("Cannot get DB version.", mlog.Err(err))
-			os.Exit(ExitGenericFailure)
-		}
+	ver, err := store.GetDbVersion(true)
+	if err != nil {
+		mlog.Critical("Cannot get DB version.", mlog.Err(err))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.